flowzone.yml

```diff
@@ -177,6 +177,76 @@ on:
         type: string
         required: false
         default: ""
+      repo_config:
+        description: "Set to true to standardise repository settings after a successful run"
+        type: boolean
+        required: false
+        default: false
+      repo_allow_forking:
+        description: "Allow forking of an organization repository"
+        type: boolean
+        required: false
+        default: true
+      repo_default_branch:
+        description: "Set the default branch name for the repository"
+        type: string
+        required: false
+        default: main
```

Comment:
Does this play nicely with ones that currently use `master`? (I assume so because github has tried to make it easy, but worth ensuring)

Reply:
We won't know until we start using it , I guess.